cabot/metricsapp/models/base.py

```diff
@@ -1,7 +1,8 @@
 from django.conf import settings
 from django.db import models
-from cabot.cabotapp.models import CHECK_TYPES, Service, StatusCheck
+from cabot.cabotapp.models import Service, StatusCheck
 from cabot.metricsapp.api import run_metrics_check
+from cabot.cabotapp.defs import *
```

Comment:
same comment about `import *`. Since there's only one field we're using `from cabot.cabotapp.defs import METRICS_STATUS...` would be better